File: dataset/output-bk/MISTRAL_codestral-latest/Task182_MISTRAL_codestral-latest.cpp
#include <regex>
#include <string>

std::string cleanPhoneNumber(const std::string& phoneNumber) {
    std::string cleanedNumber = std::regex_replace(phoneNumber, std::regex("\\D"), "");
    if (cleanedNumber.substr(0, 1) == "1") {
        cleanedNumber = cleanedNumber.substr(1);
    }
    return cleanedNumber;
}

File: dataset/output-bk/MISTRAL_codestral-latest/Task182_MISTRAL_codestral-latest.java
import java.util.regex.Pattern;

public class Main {
    public static void main(String[] args) {
        System.out.println(cleanPhoneNumber("+1 (613)-995-0253"));
    }

    public static String cleanPhoneNumber(String phoneNumber) {
        String cleanedNumber = phoneNumber.replaceAll("\\D", "");
        if (cleanedNumber.startsWith("1")) {
            cleanedNumber = cleanedNumber.substring(1);
        }
        return cleanedNumber;
    }
}